web/elm/src/SideBar/Styles.elm

```diff
@@ -279,19 +279,14 @@ hamburgerMenu { isSideBarOpen, isClickable } =
 
         else
             "default"
-    , style "background-color" <|
-        if isSideBarOpen then
-            Colors.sideBarBackground
-
-        else
-            Colors.hamburgerClosedBackground
+    , style "background-color" Colors.sideBarIconBackground
     ]
 
 
-hamburgerIcon : { isHovered : Bool, isActive : Bool } -> List (Html.Attribute msg)
-hamburgerIcon { isHovered, isActive } =
+sideBarIcon : Bool -> List (Html.Attribute msg)
+sideBarIcon isHovered =
```

Comment:
perhaps this should have an `isClickable` (or just keep the `isActive`) for the reasons suggested above